go/kbfs/libkbfs/block_retrieval_queue.go

```diff
@@ -266,41 +267,55 @@ func (brq *blockRetrievalQueue) notifyWorker(priority int) {
 	brq.sendWork(workerCh)
 }
 
-func (brq *blockRetrievalQueue) initPrefetchStatusCacheLocked() {
+func (brq *blockRetrievalQueue) initPrefetchStatusCacheLocked() error {
 	if brq.prefetchStatusForTest != nil {
-		return
+		return nil
 	}
 	if !brq.config.IsTestMode() && brq.config.Mode().Type() != InitSingleOp {
 		// If the disk block cache directory can't be accessed due to
 		// permission errors (happens sometimes on iOS for some
 		// reason), we might need to rely on this in-memory map.
-		// TODO(KBFS-3750): make it an LRU cache.
 		brq.log.Warning("No disk block cache is initialized when not testing")
 	}
 	brq.log.CDebugf(nil, "Using a local cache for prefetch status")
-	brq.prefetchStatusForTest = make(map[kbfsblock.ID]PrefetchStatus)
+	// TODO: choose a value for this size
+	var err error
+	brq.prefetchStatusForTest, err = lru.New(1000)
```

Comment:
Slight preference to assigning the new cache to a temp variable first, and only assigning it to `brq.prefetchStatusForTest` once we check the `err` is `nil`.  Just to guard against `lru.New()` returning some non-`nil` broken cache in the event of an error.